src/Microsoft.Framework.DependencyInjection.Interfaces/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.Framework.DependencyInjection.Tests")]
```

Comment:
yeah we have the same on `Microsoft.Framework.DependencyInjection` assembly as well.